src/docquery/ext/model.py

```diff
@@ -1,27 +1,34 @@
-# NOTE: This code is currently under review for inclusion in the main
-# huggingface/transformers repository:
-# https://github.com/huggingface/transformers/pull/18407
-""" PyTorch LayoutLM model."""
-
-
-import math
+from dataclasses import dataclass
 from typing import Optional, Tuple, Union
 
 import torch
 from torch import nn
-from torch.nn import CrossEntropyLoss
-from transformers.modeling_outputs import QuestionAnsweringModelOutput
-from transformers.models.layoutlm import LayoutLMModel, LayoutLMPreTrainedModel
+from torch.nn import CrossEntropyLoss, Linear
```

Comment:
nit: We use `nn.Linear` to within this file too. Perhaps we can just use `nn.CrossEntropyLoss` and `nn.Linear` and avoid the separate import?

Reply:
Good idea, fixed.